jwst/outlier_detection/utils.py

```diff
@@ -30,8 +31,104 @@ def create_cube_median(cube_model, maskpt):
     return median
 
 
+def create_median_library(resampled_models, maskpt, on_disk=True, buffer_size=10.0):
```

Comment:
create_median() and compute_weight_threshold_container() (but NOT _create_median()) are now unused and should be removed.  check for other utils functions that are not used

Reply:
perhaps remove the _library from the function definitions here too, since this will be the only median computation left